safe_core/src/client/mock/vault.rs

```diff
@@ -408,10 +411,10 @@ impl Vault {
 
             match request_type {
                 RequestType::PrivateGet | RequestType::Mutation | RequestType::Transaction => {
+                    let account = self.get_account(&requester.name());
                     // For apps, check if its public key is listed as an auth key.
                     if is_app {
-                        let auth_keys = self
-                            .get_account(&requester.name())
+                        let auth_keys = &account
```

Comment:
We don't seem to use the `account` variable elsewhere, could be left unchanged as it was before.